### PR TITLE
Zero Downtime Migration 3/4

### PR DESCRIPTION
**Pull request by jalcine**
What: This adds more checks around the database's stored value of `name`.

### CODE REVIEW

**Comment by jalcine** [comment]
Blocked by #4.

**Comment by jalcine** [comment]
CC: @pallavi

File: db/migrations/20171127145425_back_fille_movies_name.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+exports.up = function (knex, Promise) {
```
Comment (pallavi) [inline db/migrations/20171127145425_back_fille_movies_name.js]: nit: use arrow functions
nit: capitalize Knex
nit: remove unused arguments (Promise here and Knex below)

File: db/migrations/20171127145425_back_fill_movies_name.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+'use strict';
+
+exports.up = (Knex) => Knex.raw('UPDATE movies SET name = title');
```
Comment (jesscxu) [inline db/migrations/20171127145425_back_fill_movies_name.js]: i think normally there is a new line that separates these two, but lgtm otherwise

**Pull request merged**